collect_app/src/main/java/org/odk/collect/android/utilities/FormEntryPromptUtils.java

```diff
@@ -37,6 +40,18 @@ public static String getAnswerText(FormEntryPrompt fep, Context context) {
         IAnswerData data = fep.getAnswerValue();
         final String appearance = fep.getQuestion().getAppearanceAttr();
 
+        if (data instanceof SelectMultiData) {
+            String text;
+            StringBuilder b = new StringBuilder();
+            List<Selection> values = (List<Selection>) data.getValue();
+            for (Selection value : values) {
+                b.append(fep.getSelectItemText(value)).append(", ");
+            }
+            b.delete(b.length() - 2, b.length());
+            text = b.toString();
+            return text;
```

Comment:
you can just return `b.toString()` then you don't need `text` at all.

Reply:
Done